docs/source/api/user_current_jobs.rst

```diff
@@ -122,7 +122,11 @@ Creates a new content revalidation job.
 .. Note:: This method forces a HTTP *revalidation* of the content, and not a new ``GET`` - the origin needs to support revalidation according to the HTTP/1.1 specification, and send a ``200 OK`` or ``304 Not Modified`` HTTP response as appropriate.
 
 :Auth. Required: Yes
-:Roles Required: "admin" or "operations"\ [1]_
+:Roles Required: "portal"
+
+	.. versionchanged:: ATCv3.0.2
```

Comment:
need to figure out if this is going in 3.0.2, 3.1 or 4.0

Reply:
Throwing this in as a patch and bumping the patch number seems easiest IMO.

Reply:
looks like we are going to do a 3.1. want to change this to that?

Reply:
Yep, I'm on it